[2024-11-05T07:18:56.801+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-05T07:18:56.847+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task manual__2024-11-05T07:18:25.572574+00:00 [queued]>
[2024-11-05T07:18:56.879+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_task manual__2024-11-05T07:18:25.572574+00:00 [queued]>
[2024-11-05T07:18:56.880+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-11-05T07:18:56.956+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2024-11-05 07:18:25.572574+00:00
[2024-11-05T07:18:56.967+0000] {standard_task_runner.py:72} INFO - Started process 230 to run task
[2024-11-05T07:18:56.971+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'our_first_dag_v2', 'second_task', 'manual__2024-11-05T07:18:25.572574+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/our-first-dag.py', '--cfg-path', '/tmp/tmpe2smxd3v']
[2024-11-05T07:18:56.974+0000] {standard_task_runner.py:105} INFO - Job 22: Subtask second_task
[2024-11-05T07:18:57.137+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.second_task manual__2024-11-05T07:18:25.572574+00:00 [running]> on host f4d9f79457ef
[2024-11-05T07:18:57.383+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Adithya' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-05T07:18:25.572574+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-05T07:18:25.572574+00:00'
[2024-11-05T07:18:57.385+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-05T07:18:57.432+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-11-05T07:18:57.436+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hello I am after task1']
[2024-11-05T07:18:57.454+0000] {subprocess.py:86} INFO - Output:
[2024-11-05T07:18:57.457+0000] {subprocess.py:93} INFO - hello I am after task1
[2024-11-05T07:18:57.458+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-11-05T07:18:57.555+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-05T07:18:57.556+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_task, run_id=manual__2024-11-05T07:18:25.572574+00:00, execution_date=20241105T071825, start_date=20241105T071856, end_date=20241105T071857
[2024-11-05T07:18:57.673+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-05T07:18:57.720+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-05T07:18:57.724+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
